.github/workflows/_build_llvm_libraries.yml

```diff
@@ -0,0 +1,47 @@
+# CALLED WORKFLOW
```

Comment:
Should add license header like other yml files.

Rename `_build_llvm_libraries.yml` to `build_llvm_libraries.yml`